code/modules/clothing/factions/srm.dm

```diff
@@ -0,0 +1,154 @@
+//////////////
+//Jumpsuits//
+/////////////
+
+/obj/item/clothing/under/suit/roumain
+	name = "saint-roumain's worksuit"
+	desc = "A simple, hard-wearing suit designed for the hardworking hunters of the Saint-Roumain Militia."
+	icon_state = "rouma_work"
+	item_state = "rouma_work"
+	can_adjust = FALSE
+	icon = 'icons/obj/clothing/faction/srm/uniforms.dmi'
+	mob_overlay_icon = 'icons/mob/clothing/faction/srm/uniforms.dmi'
+
+//////////////////
+//Armored suits//
+/////////////////
+
+/obj/item/clothing/suit/armor/roumain
+	name = "saint-roumain duster"
+	desc = "A coat made from hard leather. Meant to withstand long hunts in harsh wilderness."
+	icon_state = "armor_rouma"
+	item_state = "rouma_coat"
+	body_parts_covered = CHEST|GROIN|ARMS
+	cold_protection = CHEST|GROIN|ARMS
+	heat_protection = CHEST|GROIN|ARMS
+	icon = 'icons/obj/clothing/faction/srm/suits.dmi'
+	mob_overlay_icon = 'icons/mob/clothing/faction/srm/suits.dmi'
+
+/obj/item/clothing/suit/armor/roumain/shadow
+	name = "saint-roumain shadow duster"
+	desc = "A coat made from hard leather. Its rough, barely-treated finish is typical of one of the Saint-Roumain Militia's trainees."
+	icon_state = "armor_rouma_shadow"
+	item_state = "rouma_shadow_coat"
+
+/obj/item/clothing/suit/toggle/labcoat/roumain_med
+	name = "saint-roumain medical duster"
+	desc = "A coat made from hard leather and further treated with exotic sterilizing oils and wax. The treatment and its more closed design offers much better protection against biological hazards."
+	icon = 'icons/obj/clothing/faction/srm/suits.dmi'
+	mob_overlay_icon = 'icons/mob/clothing/faction/srm/suits.dmi'
+	icon_state = "rouma_med_coat"
+	armor = list("melee" = 35, "bullet" = 30, "laser" = 30, "energy" = 40, "bomb" = 25, "bio" = 50, "rad" = 0, "fire" = 50, "acid" = 50)
+
+/obj/item/clothing/suit/hazardvest/roumain
+	name = "saint-roumain machinist leather vest"
+	desc = "A modified Roumain leather duster with its large flaps and sleeves cut off to provide extra mobility when maintaining weapons and vessels belonging to the Church of Saint Roumain. Its specialty treatment grants it better protection against acid and fire."
+	icon = 'icons/obj/clothing/faction/srm/suits.dmi'
+	mob_overlay_icon = 'icons/mob/clothing/faction/srm/suits.dmi'
+	icon_state = "armor_rouma_machinist"
+	item_state = "rouma_coat"
+	armor = list("melee" = 30, "bullet" = 20, "laser" = 20, "energy" = 40, "bomb" = 35, "bio" = 0, "rad" = 0, "fire" = 60, "acid" = 60)
+
+/obj/item/clothing/suit/armor/roumain/flamebearer
+	name = "saint-roumain flambearer robes"
+	desc = "A set of ashy-grey robes made from hard leather, adorned with gold trims. Its rough finish after a near-char and application of aromatics is heavily favored for the ecclesiastical sect of the Church of Saint Roumain, a living reminder of the Ashen Huntsman himself."
```

Comment:
"A set of ash-grey robes made from hard leather and adorned with gold trim and impregnated with aromatic oils. The rough, nearly-charred finish is a favorite of the ecclesiastical sect of the Church of Saint Roumain, a constant reminder of the Ashen Huntsman himself."